README.md

```diff
@@ -176,14 +199,40 @@ func ValidateWebhook(httpRequest *http.Request) (bool) {
 ```
 
 ## Develop and test
+`docker` and `docker-compose` must be installed
+
+### mac/linux with make
+Using the make file is the easiest way to get started with the tests and wraps the manual steps below with easy to use
+make commands.
 
-There's nothing special to note about the tests except that if you have Docker
-and Compose installed, you can test like this:
+```shell
+make && make test
+```
+#### Makefile goals
+* `make` or `make container`: default goal is to make the `go-shopify:latest` build container
+* `make test`: run go test in the container
+* `make clean`: deletes the `go-shopify:latest` image and coverage output
+* `make coverage`: generates the coverage.html and opens it
+
+### windows and manually
```

Comment:
windows -> Windows.

Reply:
Also, there's `make` available for Windows and really, the make file lists all the commands so that could be the source of truth instead of spelling it out here again.

Reply:
ya i usually do a make.bat but i did this all at 430am and lost motivation. maybe will add